test/cpp/end2end/grpclb_end2end_test.cc

```diff
@@ -198,6 +210,12 @@ class BalancerServiceImpl : public BalancerService {
       if (serverlist_done_) goto done;
     }
     {
+      // The loadbalancer should see a dummy user agent because it was
+      // specifically configured at the client using
+      // GRPC_ARG_GRPCLB_CHANNEL_ARGS
+      auto it = context->client_metadata().find("user-agent");
+      EXPECT_TRUE(it != context->client_metadata().end());
```

Comment:
This one should be `ASSERT_TRUE()`, because if the expectation is false, the next line will cause a crash.

Reply:
Replaced with GPR_ASSERT since ASSERT_TRUE could only be used in functions returning void.